src/inputs/dropdownButton/dropdownButton.jsx

```diff
@@ -16,18 +19,66 @@ import Icon from '../../dataDisplay/icon';
 import makeStyles from '../../styles/makeStyles';
 
 const propTypes = {
+    /**
+     * Dropdown Menu options
+     */
+    children: PropTypes.node.isRequired,
+    /**
+     * Additional classes.
+     */
+    className: PropTypes.string,
     /**
      * Override or extend the styles applied to ButtonDropdown.
      */
     classes: PropTypes.shape({
         dropdownMenuRoot: PropTypes.string,
         root: PropTypes.string,
     }),
+    /**
+     * Color of the Dropdown Button.
+     */
+    color: PropTypes.oneOf([
+        'active',
+        'default',
+        'error',
+        'link',
+        'primary',
+        'secondary',
+        'success',
+        'warning',
+    ]),
+    /**
+     * A Dropdown Button can reduce its padding.
+     */
+    compact: PropTypes.bool,
     /**
      * Dependent on the designVersion number, the component can either use our old Button component
      * or the new Button component.
      */
     designVersion: PropTypes.number,
+    /**
+     * Deprecated prop. Please use `disabled` instead.
+     */
+    disable: PropTypes.bool,
+    /**
+     * A Dropdown Button can be disabled.
+     */
+    disabled: PropTypes.bool,
+    /**
+     * The Dropown Button will be resized to its parent container's width. (Using v1 Button)
+     */
+    fluid: PropTypes.bool,
+    /**
+     * The Dropown Button will be resized to its parent container's width. (Using v2 Button)
+     */
+    fullWidth: PropTypes.bool,
+    /**
+     * ??? I don't know what to put here because I don't fully understand what this does =)
+     */
+    icon: PropTypes.bool,
```

Comment:
Señor @cbrewer1689 ... could you help a brother out here? ;-)

Reply:
Fixed

Reply:
Lookin' good.  Thanks!  https://github.com/saddlebackdev/react-cm-ui/pull/362/commits/894ed34061c181a2ef1103ca6bd6320635911475#diff-484bd1d32d50baad395b1f4b4222738641cbac745957095adb3859a0bb833788L76-R76